tests/harness/verify_proofs.py

```
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+"""
+Minimal proof verifier skeleton for the Ledger‑Kernel compliance harness.
```

Ruff (0.14.2)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)